### PR TITLE
Add more detail to Upgrading NFS Ganesha section

### PR DESCRIPTION
**Pull request by tserong**
Fixes: https://github.com/SUSE/doc-ses/issues/872
Signed-off-by: Tim Serong <tserong@suse.com>

### CODE REVIEW

File: xml/admin_ceph_upgrade.xml [hunk 1]
```diff
@@ -830,6 +839,13 @@ drwx------ 9 root root 4.0K Sep 8 03:23 ..
 NAME           HOST   STATUS         REFRESHED  AGE  VERSION  IMAGE NAME                                IMAGE ID      CONTAINER ID
 nfs.foo.node2  node2  running (26m)  8m ago     27m  3.3      registry.suse.com/ses/7/ceph/ceph:latest  8b4be7c42abd  c8b75d7c8f0d</screen>
     </step>
+    <step>
+     <para>
+      Repeat these steps for each &ganesha; node. You don't need to create a separate
```
Comment (lproven) [inline xml/admin_ceph_upgrade.xml:844]:
```suggestion
      Repeat these steps for each &ganesha; node. You do not need to create a separate
```

File: xml/admin_ceph_upgrade.xml [hunk 2]
```diff
@@ -830,6 +839,13 @@ drwx------ 9 root root 4.0K Sep 8 03:23 ..
...
+      service specification for each node, it's enough to add each node's hostname to
```
Comment (lproven) [inline xml/admin_ceph_upgrade.xml:845]:
```suggestion
      service specification for each node. It is sufficient to add each node's hostname to
```

**Review by lproven — CHANGES_REQUESTED** [review]

**Review by mgfritch — APPROVED** [review]
thanks for catching this! looks like some of my incorrect dev notes somehow landed in the product doc ;)

lgtm, minus the suggestions from @lproven

**Review by tbazant — APPROVED** [review]
great work with docbook :-)

**Pull request merged**